### ISSUE TITLE
Installation error with 2.1.0

### ISSUE THREAD

**Issue by heckfi** [pinterest/pymemcache#214]
```Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-hpcqy0cc/pymemcache/setup.py", line 5, in <module>
        from pymemcache import __version__
      File "/tmp/pip-install-hpcqy0cc/pymemcache/pymemcache/__init__.py", line 3, in <module>
        from pymemcache.client.base import Client  # noqa
      File "/tmp/pip-install-hpcqy0cc/pymemcache/pymemcache/client/__init__.py", line 3, in <module>
        from pymemcache.client.base import Client  # noqa
      File "/tmp/pip-install-hpcqy0cc/pymemcache/pymemcache/client/base.py", line 16, in <module>
        import six
    ModuleNotFoundError: No module named 'six'`

**Comment by heckfi** [comment]
> That's strange. We include `install_requires=['six']` in the `setup.py` definition.
> 
> Can you share more of the installation log in case there's an earlier error we're missing?

yes, but `from pymemcache import __version__` import module, where use six .

`Downloading https://files.pythonhosted.org/packages/47/be/277821fae9a50c66f2246617be211d0696d6cbbb74495d00d28d9014f316/pymemcache-2.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-y7yfiuyz/pymemcache/setup.py", line 5, in <module>
        from pymemcache import __version__
      File "/tmp/pip-install-y7yfiuyz/pymemcache/pymemcache/__init__.py", line 3, in <module>
        from pymemcache.client.base import Client  # noqa
      File "/tmp/pip-install-y7yfiuyz/pymemcache/pymemcache/client/__init__.py", line 3, in <module>
        from pymemcache.client.base import Client  # noqa
      File "/tmp/pip-install-y7yfiuyz/pymemcache/pymemcache/client/base.py", line 16, in <module>
        import six
    ModuleNotFoundError: No module named 'six'`

**Comment by methane** [comment]
FYI, https://packaging.python.org/guides/single-sourcing-package-version/

**Comment by jparise** [comment]
Ah, I see now. This was an unintended consequence of #197.

I'll address this in the next few days and get a point release out with the fix.

**Comment by jparise** [comment]
I just released [version 2.1.1](https://github.com/pinterest/pymemcache/releases/tag/v2.1.1) with a fix for this issue.
